backend/src/main/java/org/eclipse/tractusx/semantics/registry/security/AuthorizationEvaluator.java

```diff
@@ -20,12 +20,8 @@
 package org.eclipse.tractusx.semantics.registry.security;
```

Comment:
I think you forgot to adjust the copyright headers. you can check for other files also.

Reply:
done